README.md

```diff
@@ -1,2 +1,169 @@
-# portfolio_project
-This is almost a clone of 1st week of portfolio assignment with semantic tags of HTML -> to review by my own.
```

Comment:
This should be proper under description section

Reply:
- [ ] Good work using Github inline features to perform the review. Please make sure the review meets each of the following.
   -  The comments should be polite. You can offer them as suggestions instead of commands ✔️ 
   -  Include an explanation as to why the change is necessary ✔️ 
   -  The comment should be clear and easy to understand ✔️

Reply:
Please kindly state the reason(s) for the required change in a more friendly and polite manner. It might be obvious to you and not so for others. 👍

Reply:
@BilalLiaquat7 please take note that both reviewers @wayungi & @OyePriscilla requested these changes but you have stitll not implemented them. what you have been requested to do by your two previous reviewers is to give a detailed explanation of the change you want the student to make and provide reasons why you are requesting these changes. currently the below text is what you have written
- This should be proper under description section
The above text does not meet the project requirements you have not used polite language to request for the change such as `i kindly suggest, i think you can, please ensure you`. you have also not provided a good reason why the student should make the change. the below example is how you should write a review .
- example review message
- [ ] I want to kindly suggest that you make use of a proper description for your project, it is important you carefully describe your project accordingly to what you have built so that anyone accessing your website has a fully knowledge of what your project is all bout. :+1:
- @BilalLiaquat7 please also note that you are to write at least two reviews for this project requirement, this was mentioned by first reviewer, kindly add new another review to meet this requirement before submitting. :+1:

Reply:
> @BilalLiaquat7 please take note that both reviewers requested these changes but you have still not implemented them correctly. what you have been requested to do by your two previous reviewers is to give a detailed explanation of the change you want the student to make and provide reasons why you are requesting these changes. currently the below text is what you have written
> 
> * This should be proper under description section
>   The above text does not meet the project requirements you have not used polite language to request for the change such as `i kindly suggest, i think you can, please ensure you`. you have also not provided a good reason why the student should make the change. the below example is how you should write a review .
> * > example review message
> * [x]  I want to kindly suggest that you make use of a proper description for your project, it is important you carefully describe your project accordingly to what you have built so that anyone accessing your website has a fully knowledge of what your project is all bout. 👍
> * @BilalLiaquat7 please also note that you are to write at least two reviews for this project requirement, this was mentioned by first reviewer, kindly add new another review to meet this requirement before submitting. 👍 ✨

- [ ] Please follow suit what the reviewer has suggested above .. for the number of reviews seems you did more than one, it has just been marked as resolved. So that other reviewers don't think the same, kindly add it again.